NetCoreRestApi/DataLayer/Repositories/Interfaces/IProductRepository.cs

```diff
@@ -0,0 +1,8 @@
+﻿using DataLayer.Models;
+
+namespace DataLayer.Repositories
+{
+    public interface IProductRepository<TId> : IRepository<ProductModel<TId>, TId>
```

Comment:
Fix to interface IProductRepository : IRepository<ProductModel, TId>

Reply:
done